frontend/packages/knative-plugin/src/components/functions/GettingStartedCard.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
```

Comment:
Is this component really used? If not please delete it together with the scss file.